sunpy/net/dataretriever/tests/test_goes_ud.py

```diff
@@ -15,9 +19,72 @@
 from sunpy.time import parse_time, is_time_equal
 
 
-@pytest.fixture
-def LCClient():
-    return goes.XRSClient()
+LCClient = goes.XRSClient()
+
+
+def mock_query_object(start_date, end_date):
```

Comment:
Isn't this a lot of duplicated code from other PRs? Is there a better way to do this?

Reply:
I know that this is duplicated, but there are few steps which require different processing :smile: